syntax = "proto3";

package order;

service OrderService {
    rpc GetOrder (OrderRequest) returns (OrderResponse);
}

message OrderRequest {
    int32 order_id = 1;
    int32 user_id = 2;
    int32 quantity = 3;
}

message OrderResponse {
    int32 order_id = 1;
    string product = 2;
    int32 quantity = 3;
    int32 price = 4;
    string user_name = 5;
    string user_email = 6;
}